File: eclipse/idm.project.tests/ressources/expected/r/print.R
# Creates a new DataFrame from a CSV File
df = data.frame()
#Adds a new Column to the dataframe
df[1,"Stocks"] <- "Default"

#inserts a value at a cell
df[0+1,"Stocks"] = (10+10)

#Prints a expression
print(as.numeric(df[0+1,"Stocks"]))

write.csv(df,"C:/Users/Leloup/Documents/FAC/M2/IDM/Git/IDM-Projet/eclipse/idm.project.tests/ressources/output/printR.csv",quote = FALSE,row.names = FALSE)

File: eclipse/idm.project.tests/ressources/expected/python/print.py
import pandas as pd
df = pd.DataFrame(list())
df.insert(0,"Stocks",0)
df.at[0, "Stocks"] = (10+10)
print(df.at[0,"Stocks"])
df.to_csv("/home/simonlecordier/workspace/m1/IDM/IDM-Projet/eclipse/idm.project.tests/ressources/output/printPython.csv", index=False)

